module/zfs/arc.c

```diff
@@ -7865,6 +7971,22 @@ arc_init(void)
 
 	arc_prune_taskq = taskq_create("arc_prune", zfs_arc_prune_task_threads,
 	    defclsyspri, 100, INT_MAX, TASKQ_PREPOPULATE | TASKQ_DYNAMIC);
+	if (likely(max_ncpus > 1)) {
```

Comment:
It does not really matter, but `likely` specifier here and below for code executed only once and which performance does not matter only pollutes the code.

Reply:
Removed.